edb/graphql/types.py

```diff
@@ -299,6 +299,14 @@ def _convert_edb_type(self, edb_target):
             if name in self._gql_enums:
                 target = self._gql_enums.get(name)
 
+        elif edb_target.is_tuple():
+            edb_typename = edb_target.get_verbosename(self.edb_schema)
+            raise g_errors.GraphQLCoreError(
+                f"Could not convert {edb_typename} to"
+                f" a GraphQL type. "
```

Comment:
Again, I'd just say that tuples are not currently supported without elaborating.

Reply:
Fair enough to remove the extra details from this message.